goldstone/core/serializers.py

```diff
@@ -98,3 +98,45 @@ def to_representation(self, instance):
         """Return an already-serialized object."""
 
         return instance
+
+
+class EventSerializer(ReadOnlyElasticSerializer):
+    """Serializer for event data that's stored in ElasticSearch."""
+
+    class Meta:        # pylint: disable=C1001,W0232
+        """Exclude several uninteresting fields."""
+
+        exclude = ('@version', 'message', 'syslog_ts', 'received_at', 'sort',
```

Comment:
These are legacy.  This tuple should be trimmed to contain only fields relevant to events in the current events index.